docs/recipes/Authentication.md

```diff
@@ -0,0 +1,1447 @@
+---
+title: Authentication with Supabase
+description: Here we use Supabase as our backend to show out to implement authentication with your React Native project.
```

Comment:
Maybe just me but this sounds a big weird no? 

`We use Supabase as our backend to demonstrate how to implement authentication in your React Native project.`

Reply:
https://github.com/infinitered/ignite-cookbook/pull/158/commits/fc2b64e4a7b2f51b9a045be68aa7f82d66142673

Let me know how that lands.